assets/js/data/cart/actions.js

```diff
@@ -242,3 +242,66 @@ export function* changeCartItemQuantity( cartItemKey, quantity ) {
 
 	yield itemQuantityPending( cartItemKey, false );
 }
+
+/**
+ * Selects a shipping rate.
+ *
+ * @param {string} rateId the id of the rate being selected.
+ * @param {number} packageId the key of the packages that we will select within.
```

Comment:
Not a big deal, but this can be also marked as optional. :slightly_smiling_face: 
```suggestion
 * @param {number} [packageId] the key of the packages that we will select within.
```